### DISCUSSION TITLE
tui helptext: Use extended characters for help.

### DISCUSSION THREAD

**Discussion by jtagcat** [dundee/gdu#68] [Ideas]
I'd say to not replace `enter` with `⏎`, but `enter ⏎`, having both could be an idea?

**Comment by dundee** [comment]
![extended](https://user-images.githubusercontent.com/27106/126454275-9e021b7e-c2a7-42a4-ac81-00a2d6bbdcf5.png)
![original](https://user-images.githubusercontent.com/27106/126454279-eacbc118-c67e-4234-9df6-99cf90116baa.png)

I have tried it, but not sure if it's more readable now. What do you think?

**Reply by dundee** [reply]
https://github.com/dundee/gdu/pull/73

**Reply by jtagcat** [reply]
yeah it looks worse

**Reply by dundee** [reply]
ok, I will close the PR.
